opentracing/tracecontext.go

```diff
@@ -51,18 +43,18 @@ type TraceContext interface {
 	// `restrictedKey` MUST match the regular expression
 	// `(?i:[a-z0-9][-a-z0-9]*)` and is case-insensitive. That is, it must
 	// start with a letter or number, and the remaining characters must be
-	// letters, numbers, or hyphens. See CanonicalizeTraceTagKey(). If
-	// `restrictedKey` does not meet these criteria, SetTraceTag() results in
-	// undefined behavior.
+	// letters, numbers, or hyphens. See CanonicalizeTraceAttributeKey(). If
+	// `restrictedKey` does not meet these criteria, SetTraceAttribute()
+	// results in undefined behavior.
 	//
 	// Returns a reference to this TraceContext for chaining, etc.
-	SetTraceTag(restrictedKey, value string) TraceContext
+	SetTraceAttribute(restrictedKey, value string) TraceContext
```

Comment:
on the fence about calling this `SetTraceAttribute` or just `SetAttribute`... went for the more explicit thing since this – while powerful – will not be called frequently.